modules/utilities.py

```diff
@@ -56,6 +56,7 @@ def verifier(*args, **kwargs):
             if token not in allowed_tokens:
                 current.response.status = 401
                 return current.response
+            current.request.extension = 'json'
```

Comment:
why is this required?

Reply:
the API requests need to always return JSON response
writing this here will save us from appending the .json extension in the request urls